StreamDeckSDK/ESDSDKDefines.h

```diff
@@ -159,5 +163,6 @@ enum {
   kESDSDKDeviceType_StreamDeckXL = 2,
   kESDSDKDeviceType_StreamDeckMobile = 3,
   kESDSDKDeviceType_CorsairGKeys = 4,
-  kESDSDKDeviceType_StreamDeckPedal = 5
+  kESDSDKDeviceType_StreamDeckPedal = 5,
+  kESDSDKDeviceType_StreamDeckPlus = 7 // From volume controller manifest.json
```

Comment:
For my reference, 6 should be kESDSDKDeviceType_CorsairVoyager

Reply:
I see, I will add that.